app/charts/shared/legend-color.tsx

```diff
@@ -142,7 +149,7 @@ const useLegendGroups = ({
   const segmentValues =
     segmentFilters?.type === "multi" ? segmentFilters.values : emptyObj;
 
-  return useMemo(() => {
+  return React.useMemo(() => {
```

Comment:
Ah we should settle on one style 😁 I personally do not like to see the code full of "React", is there a specific reason why you like this style ?

Reply:
I remember that in the past we needed to `import React from "react"` anyway, so having `import React, { useMemo } from "react"` felt like doing the same thing twice for me, but it looks like [since React 17 it's not needed anymore](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform) 😁

I am fine with using named imports, maybe we can then have an eslint rule, similar to lodash and d3 to enforce this?

Reply:
OK, I can do that. It is fine in this PR, there will be a refactor PR.